web/src/public/index.js

```diff
@@ -119,52 +74,167 @@ function plotTravelMonths(select, onSelect) {
         .text(d => d);
 }
 
-function plotParks(svg, npd, projection, filters) {
-    const defaultRadius = 3.5;
-    const defaultColor = '#0026ff';
+function plotParks(svg, npd, scores, projection, filters) {
+    const defaultColor = '#4682b4';
+    const parkRadius = d3.scaleLog().range([2, 8]).domain(d3.extent(npd, d => d.area));
+    let tooltip = d3.select("body").append("div") // for image
+        .attr("class", "tooltip")
+        .style("opacity", 1e-6);
 
-    svg.selectAll('circle').remove();
-    svg.selectAll('circle')
+    let parks = svg.selectAll("circle")
         .data(npd)
         .enter()
-        .append('circle')
-        .attr('cx', d => projection([d.long, d.lat])[0])
-        .attr('cy', d => projection([d.long, d.lat])[1])
-        .attr('r', defaultRadius) // todo: Update size based on park ranking
-        .attr('fill', defaultColor)
-        .on('mouseover', (val, idx, array) => {
-            console.log(val);
-            const circle = array[idx];
-            d3.select(circle)
-                .attr('r', defaultRadius * 2)
-            // .attr('fill', 'red');
+        .append("circle")
+        .attr("r",  d => parkRadius(d.area))
+        .attr("transform", function(d) {
+            return "translate(" + projection([
+            d.long,
+            d.lat
+            ]) + ")";
```

Comment:
I think something might be up here with this. Like I said in slack, I'm seeing really odd/broken behavior with the circles. Why was the change made to use translate instead of the cx and cy that was there before?